src/Akka.HealthCheck.Persistence/AkkaPersistenceLivenessProbe.cs

```diff
@@ -156,20 +168,35 @@ private bool HandleMessages(object message)
                     _probe = null;
                     if(_logInfo)
                         _log.Debug($"Persistence probe terminated. Recreating in {_delay.TotalSeconds} seconds.");
+                    
+                    Timers.CancelAll();
                     ScheduleProbeRestart();
                     return true;
                 
                 case CreateProbe:
                     if(_logInfo)
                         _log.Debug("Recreating persistence probe.");
                     
+                    Timers.StartSingleTimer(TimeoutTimerKey, CheckTimeout.Instance, _timeout);
```

Comment:
Guarding against suicide actor deadlock

Reply:
We're doing it here inside the probe actor instead of inside the suicide actor because mailbox processing can be blocked inside a persistence actor, making timers unreliable.

Reply:
Good call